### ISSUE TITLE
convert tf mode error, "Check failed: channels > 0 ==> Batchnorm Channels Paramter is Wrong"

### ISSUE THREAD

**Issue by diligent321** [alibaba/MNN#159]
hello, a FusedBatchNorm layer with only one channel is included in my network, as the following image indicated. 

![61100371-83849800-a498-11e9-9114-079269dd95cf](https://user-images.githubusercontent.com/31379237/61178879-d5a8f300-a629-11e9-8f7c-872ab8920b08.PNG)

By the way, it's a segmentation task, so when I run MNNConvert command, it raised an error like this,

"MNN/tools/converter/source/tensorflow/FusedBatchNormTf.cpp:46: Check failed: channels > 0 ==> Batchnorm Channels Paramter is Wrong! 0 "

how to solve the problem?

**Comment by czy2014hust** [comment]
> hello, a FusedBatchNorm layer with only one channel is included in my network, as the following image indicated.
> 
> ![61100371-83849800-a498-11e9-9114-079269dd95cf](https://user-images.githubusercontent.com/31379237/61178879-d5a8f300-a629-11e9-8f7c-872ab8920b08.PNG)
> 
> By the way, it's a segmentation task, so when I run MNNConvert command, it raised an error like this,
> 
> "MNN/tools/converter/source/tensorflow/FusedBatchNormTf.cpp:46: Check failed: channels > 0 ==> Batchnorm Channels Paramter is Wrong! 0 "
> 
> how to solve the problem?

Could you please send your model to me?

**Comment by diligent321** [comment]
https://github.com/fengjian0106/hed-tutorial-for-document-scanning/raw/master/checkpoint/hed_graph.pb  @czy2014hust

**Comment by czy2014hust** [comment]
fixed!
